scripts/rfsuite/widgets/status/status.lua

```diff
@@ -3146,6 +3146,7 @@ function status.getSensors()
                 current = currentSOURCE:value()
                 if currentSOURCEESC1 ~= nil then
                         currentesc1 = currentSOURCEESC1:value()
+                        if currentesc1 == nil then currentesc1 = 0 end
```

Comment:
Hint: you could write
```
currentesc1 = currentSOURCEESC1:value() or 0
```

Reply:
Yes.  I recently discovered this syntax!